internal/app/init/pkg/system/services/kubelet.go

```diff
@@ -87,11 +105,12 @@ func (k *Kubelet) Start(data *userdata.UserData) error {
 		{Type: "cgroup", Destination: "/sys/fs/cgroup", Options: []string{"rbind", "rshared", "rw"}},
 		{Type: "bind", Destination: "/dev", Source: "/dev", Options: []string{"rbind", "rshared", "rw"}},
 		{Type: "bind", Destination: "/var/run", Source: "/run", Options: []string{"rbind", "rshared", "rw"}},
-		{Type: "bind", Destination: "/var/lib/kubelet", Source: "/var/lib/kubelet", Options: []string{"rbind", "rshared", "rw"}},
-		{Type: "bind", Destination: "/var/lib/containerd", Source: "/var/lib/containerd", Options: []string{"rbind", "rshared", "rw"}},
+		// 'blind' Support for calico / containerd / kubelet / rook(?)
+		//  or should we be very explicit here?
```

Comment:
Currently we favor explicit:
https://github.com/autonomy/talos/blob/ee226dddac3bd2a631093b44e05ce4725a39ed95/internal/app/init/internal/rootfs/cni/cni.go

I have been torn between the convenience of exposing a wider part of the root filesystem vs maintaining an explicit list, but I think since the principle of least privilege is something we have strived for in other parts of the code that maybe we should stick with explicit for now.